extundelete: add page (#4327)
tldr-pages/tldr: pages/linux/extundelete.md

# extundelete

> > Restore files from an unmounted ext3 or ext4 partition.

Reviewer: Maybe this is more clear:
```suggestion
> Recover deleted files from ext2, ext3 or ext4 partitions by parsing the journal.
```

Author: I will add the rest of the suggestion, but this utility does not work with ext2 filesystems as they lack a journal.
```suggestion
> Recover deleted files from ext3 or ext4 partitions by parsing the journal.
```

Reviewer: 😮 so, Arch Linux has a bug in their package description https://www.archlinux.org/packages/community/x86_64/extundelete/

> See also `date` for Unix time information and `umount` for unmounting partitions.

> > More information: <http://extundelete.sourceforge.net/>.

Reviewer: ```suggestion
> More information: <http://extundelete.sourceforge.net>.
```

> - Restore all deleted files inside partion 1 on device b (`sdb1`):
> 
> `sudo extundelete {{/dev/sdb1}} --restore-all`

Reviewer: ```suggestion
- Restore all deleted files inside partition N on device X:

`sudo extundelete {{/dev/sdXN}} --restore-all`
```

maybe we can remove `N on device X`, I think only advanced users use this app.

- Restore a file from a path relative to root (Do not start the path with `/`):

> `sudo extundelete {{/dev/sdb1}} --restore-file {{path/to/file}}`

Reviewer: ```suggestion
`extundelete {{/dev/sdXN}} --restore-file {{path/to/file}}`
```

- Restore a directory from a path relative to root (Do not start the path with `/`):

> `sudo extundelete {{/dev/sdb1}} --restore-directory {{path/to/directory}}`

Reviewer: ```suggestion
`extundelete {{/dev/sdXN}} --restore-directory {{path/to/directory}}`
```

- Restore all files deleted after January 1st, 2020 (in Unix time):

> `sudo extundelete {{/dev/sdb1}} --restore-all --after {{1577840400}}`

Reviewer: ```suggestion
`extundelete {{/dev/sdXN}} --restore-all --after {{1577840400}}`
```

Based on the `date` page, looks like we don't template the Unix time variable,
